dream.opam

```diff
@@ -71,6 +71,7 @@ depends: [
   "ssl" {>= "0.5.8"}  # Ssl.get_negotiated_alpn_protocol.
   "uri" {>= "4.2.0"}
   "yojson"  # ...
+  "ptime"  # time and date
```

Comment:
`Ptime.weekday` appears to [require Ptime 0.8.1](https://github.com/dbuenzli/ptime/blob/master/CHANGES.md#v081-2015-07-14-cambridge-uk).